TATB - 40 :  change design for confirmation dialog box

PrabhatJTDeveloper opened the pull request.

Commit:
Technical assistant : Implement header for all pages

PrabhatJTDeveloper committed:
TATB - 29 : Move inline CSS to external SCSS file

PrabhatJTDeveloper committed:
- TATB 39
    - for wraping dialog data into one config object changed the modal component ts file.
    - in modal component html file use that object .
    - and the modal is opening in signout which is in header component and skip button which is in quiz component so changed this two components.
    - add one object signoutModal in dialogData.json file.

PrabhatJTDeveloper committed:
TATB - 38 : Remove Auth service use to Auth guard for authentication

PrabhatJTDeveloper committed:
TATB - 30: When click skip button , pop up will remain even after question change
 -Give camelCase to variables
 -Fixed bug when click skip button the popup also swipes on timer end
 -Removed a console error

PrabhatJTDeveloper committed:
TATB:30 :
  -reverted chagnes
  - removed unwated code

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation dialog box.

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.html:
do code intendation properly

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.ts:
do code intendation properly

charmijtdev commented on src/app/authorization/login/login.component.ts:
why function is declared as public are we accessing in another class ? if not then please change it

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.scss:
why managing left and right with px ?

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.scss:
please use hashcode

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.scss:
please use hashcode for this

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.scss:
could you please try with border-top-0

charmijtdev commented on src/app/dialog-service/modal/modal/modal.component.scss:
why directly use for btb-close directly

PrabhatJTDeveloper committed:
TATB - 29 : Remove inline css and Remove popup box model in header section

PrabhatJTDeveloper committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-29-add-header-section-for-routing-all

PrabhatJTDeveloper committed:
TATB - 29 :  Fixed Merge Conflicts

PrabhatJTDeveloper committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-38-Remove-Auth-service-use-to-Auth-guard-for-authentication

PrabhatJTDeveloper committed:
- TATB 39
    - solved the conflicts
    - removed unused class and id from the button tag of header component html file.
    - added about Modal data to the dialogeData.json

PrabhatJTDeveloper committed:
TATB - 38 : Remove Auth service use to Auth guard for authentication changes-auth guards,login,header and question service name to chnage authentication service.

PrabhatJTDeveloper committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-40-change-design-for-confirmation-dialog-box

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation dialog box changes- modal component , login

PrabhatJTDeveloper committed:
TATB-32- fixed conflict in Add footer section in all page

PrabhatJTDeveloper committed:
TATB -35 :Show only recently completed result then create option for showing all results.
  -[BUG] Getting 2 results of same quiz
  -Change the card design of all results page
  -[BUG] Data does not remain after refresh
  -Get data of completed quiz
  -implement initial letter of username as display picture
  -implement show more button in all results

PrabhatJTDeveloper committed:
TATB-32- Fixed conflict in Redesign login and registration page

PrabhatJTDeveloper committed:
TATB - 36 - Fix the issue of save register user data

PrabhatJTDeveloper committed:
TATB - 38 : Remove Auth service use to Auth guard for authentication changes - auth guard,login component,header, authentication service.

PrabhatJTDeveloper committed:
- TATB 39
     - removed ?.length from the login component ts file.
     - removed import of dilogData.json from dialog service
     - updated modal component html file.

PrabhatJTDeveloper committed:
TATB-32- Change class name and  and remove href in Footer section

charmijtdev committed:
Store Implementing with ngrx

charmijtdev commented on src/app/authorization/login/login.component.ts:
please remove length as string will not return length

charmijtdev commented on src/app/authorization/login/login.component.ts:
could you please assign like JSON.parse(localstorage.getitem('registerUser) as string ) ?

charmijtdev committed:
Merge branch 'feature/development' into feature/TATB-29-add-header-section-for-routing-all

PrabhatJTDeveloper committed:
TATB - 29 : Registration page should contain header (#42)

* Technical assistant : Implement header for all pages

* TATB - 29 : Move inline CSS to external SCSS file

* TATB - 29 : Remove inline css and Remove popup box model in header section

* TATB - 29 :  Fixed Merge Conflicts

---------

Co-authored-by: Dipesh Gandhi <dipesh_gandhi9@gmail.com>
Co-authored-by: Charmi Chauhan <88572890+charmijtdev@users.noreply.github.com>

PrabhatJTDeveloper committed:
TATB- 33 : login page registration page flow change functionality (#41)

* TATB-05-BUG-Add-login-link-to-user-registration-form-login-form-component

* TATB-05-BUG-Add-login-link-to-user-registartion-form-login-form-component

* TATB - 33 : Login page Registration page flow change functionality

* TATB - 33 : Login page Registration page flow change functionality

* TATB - 33 : Login page Registartion page flow change functionality - changes - Login component,Registration component

* TATB - 33 : Login page Registration page flow change functionality changes - app module, registration component,toast component

* TATB - 33 : Login page Registration page flow change functinality changes - login, registration components

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation dialog box - change login and registration components

PrabhatJTDeveloper committed:
TATB - 38 : Remove Auth service use to Auth guard for authentication.  changes - result component and authentication service

PrabhatJTDeveloper committed:
TATB - 38 : Remove Auth service use to Auth guard for authentication changes.

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation dialog box changes

PrabhatJTDeveloper committed:
TATB-  38 : remove auth service use to auth guard for authentication (#43)

* TATB - 38 : Remove Auth service use to Auth guard for authentication

* TATB - 38 : Remove Auth service use to Auth guard for authentication changes-auth guards,login,header and question service name to chnage authentication service.

* TATB - 38 : Remove Auth service use to Auth guard for authentication changes - auth guard,login component,header, authentication service.

* TATB - 38 : Remove Auth service use to Auth guard for authentication.  changes - result component and authentication service

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation-dialog-box changes.

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation dialog box (#44)

* TATB - 40 : change design for confirmation dialog box.

* TATB - 40 : change design for confirmation dialog box changes- modal component , login

* TATB - 40 : change design for confirmation dialog box - change login and registration components

charmijtdev commented:
merged to feature/development

PrabhatJTDeveloper committed:
TATB-32- change class name and remove inline css in Add footer section

PrabhatJTDeveloper committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-32-Add-footer-section-in-all-page

PrabhatJTDeveloper committed:
TATB-29-Add droupdown menu in header section

PrabhatJTDeveloper committed:
- TATB 39
  - change the mapJsonData() to getQuestionData() function name in quiz component ts file and in statquiz component ts file mapJsonData() to getInstruction().

PrabhatJTDeveloper committed:
TATB - 37 : Redesign Quiz Test Component (#45)

* - TATB-37 redesign the quiz test template
  - for that change the html and scss file of the quiz component .

* - TATB 37
   - removed unnecessary id and role in the button from the html file
   - removed unwanted css in scss file

* - TATB 37
   - removed unwanted id and class from carousel in html file
   - removed the css which is also used as boostrap class.

* - TATB 37 removed unused div in html file

* - TATB 37
  - make change in condition of hightlighted text in quiz.component.html file

* - TATB 37
   - give the boostrap class and remove from the css
   - and make change in html file

* - TATB 37
   - make changes in html and scss file.

* - TATB 37
   - removed "this" from the {{this.selectedQuizType}} in the quiz component html file.

* - TATB 37
  - removed "this" from the html file

PrabhatJTDeveloper committed:
TATB-36-Add validation in login page

PrabhatJTDeveloper committed:
- TATB 39
    - removed inline css and give the extenal css
    - added the userId in the login component ts file
    - removed unnecassary line space and comma from the dialog service ts file
    - formated the code in modal component html file and also removed commented code.

PrabhatJTDeveloper committed:
- TATB 39
    - Removed extra comma from the login and header component ts file and also change the function name for opening dialog modal in header component ts file.
    - change the class name in header component scss file.
    - make change in dialogData.json file

PrabhatJTDeveloper committed:
TATB-29-Add header section  in main dashboard and Remove css Add to boostarp class

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-29-add-header-section-for-routing-all' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-29-add-header-section-for-routing-all

PrabhatJTDeveloper committed:
- TATB 47
     - have change the validation of password in login component ts file and in html also.

PrabhatJTDeveloper committed:
TATB-29 : Remove unwanted code

PrabhatJTDeveloper committed:
TATB - 39 [BUG]: Wrap Dialog Data into one config object for more readability (#46)

* - TATB 39
    - for wraping dialog data into one config object changed the modal component ts file.
    - in modal component html file use that object .
    - and the modal is opening in signout which is in header component and skip button which is in quiz component so changed this two components.
    - add one object signoutModal in dialogData.json file.

* - TATB 39
     - removed ?.length from the login component ts file.
     - removed import of dilogData.json from dialog service
     - updated modal component html file.

* - TATB 39
    - removed inline css and give the extenal css
    - added the userId in the login component ts file
    - removed unnecassary line space and comma from the dialog service ts file
    - formated the code in modal component html file and also removed commented code.

* - TATB 39
    - Removed extra comma from the login and header component ts file and also change the function name for opening dialog modal in header component ts file.
    - change the class name in header component scss file.
    - make change in dialogData.json file

charmijtdev committed:
Merge branch 'feature/development' into bug/TATB-47-validation-for-password

PrabhatJTDeveloper committed:
TATB - 38 : bug when user not login and no cookies found redirect to login page

PrabhatJTDeveloper committed:
- TATB 39
   - Modify dialogData.json
   - make code documentation in scss file.
   - as change the json file also updated the html file.

charmijtdev committed:
removed confiict

charmijtdev committed:
Merge branch 'feature/development' into bug/TATB-39-BUG-wrap-dialog-data-into-one-config-object

PrabhatJTDeveloper committed:
- TATB 47
   - make change the html and ts file of login and registration

PrabhatJTDeveloper committed:
Merge branch 'bug/TATB-47-validation-for-password' of https://github.com/Jyoti-Technosoft/technical-assistant into bug/TATB-47-validation-for-password

PrabhatJTDeveloper committed:
- TATB 46
   - add base path in tsconfig.json file
   - and change the path of imports of  all the ts file from ../ to @app or @assets

PrabhatJTDeveloper committed:
- TATB 46
    - resolve the conflicts.

PrabhatJTDeveloper committed:
TATB-36 : Change href tag to router link  chnage value and Add keydown function in login and registration page

PrabhatJTDeveloper committed:
TATB - 44 Redesign Dashboard page for quiz App

PrabhatJTDeveloper committed:
TATB-36 : Fix conflict in Redesign login and registration page

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-29-add-header-section-for-routing-all' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB Stagging : Merge TATB 29 header branch

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-32-Add-footer-section-in-all-page' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB -35 :Show only recently completed result then create option for showing all results.
    - change the way of getting result
    -remove unnecessary changes
    -add query params

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-35-create-individual-result-page-redesign-all-results-page' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB - 40 change design for confirmation dialog box

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-36-Redesign-login-and-registration-page' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB - 43 : State Management through NGRX
    -Implemented the store for login
    -implemented actions , effect , model , user for login
    -get data from login in state management

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-38-Remove-Auth-service-use-to-Auth-guard-for-authentication' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
Merge branch 'bug/TATB-39-BUG-wrap-dialog-data-into-one-config-object' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
Merge branch 'bug/TATB-46-use-annotation-base-import' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
Merge branch 'bug/TATB-47-validation-for-password' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
Merge branch 'feature/development' of https://github.com/Jyoti-Technosoft/technical-assistant into learning/store-implementing

PrabhatJTDeveloper committed:
TATB - 40 chnage design for confirmation dialog box

PrabhatJTDeveloper committed:
TATB -40-change design for confirmation-dialog-box changes-modal component and login component

PrabhatJTDeveloper committed:
TATB - 40 change design for confirmation dialog box changes modal component html file.

PrabhatJTDeveloper committed:
TATB - 44 : Redesign Dashbord page for quiz App changes- dashboard componet html file

charmijtdev committed:
-take pull of latest branch
-implemnted store for authorization

charmijtdev committed:
Merge branch 'learning/store-implementing' of https://github.com/Jyoti-Technosoft/technical-assistant into learning/store-implementing

PrabhatJTDeveloper committed:
TATB - 38 -bug - when user not login and no cookies found redirect to login page changes - app component.html

PrabhatJTDeveloper committed:
TATB - 44 - Redesign Dashboard page for quiz App changes - data.json file,style.scss

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-38-bug-when-user-not-login-and-no-cookies-found-redirect-to-login-page' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-44-Redesign-Dashboard-page-for-quiz-App' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-40-change-design-for-confirmation-dialog-box' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB - 35 : Show only recently completed result then create option for showing all results.
      -remove dialof errors
      -username not visible
      -couldnt login error

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-35-create-individual-result-page-redesign-all-results-page' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB - 40 - change design for confirmation dialog box

PrabhatJTDeveloper committed:
TATB : Fix issue rename emailId to email

charmijtdev committed:
--added store for whole registration

PrabhatJTDeveloper committed:
Merge branch 'feature/authorization-store' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/stagging-all-PR-changes

PrabhatJTDeveloper committed:
TATB : Make login page responsive

PrabhatJTDeveloper committed:
TATB : Make footer sticky and also make responsive registration page

PrabhatJTDeveloper committed:
TATB : Change header avatar icon

PrabhatJTDeveloper committed:
TATB : Add background color to footer section

PrabhatJTDeveloper committed:
TATB : Malke confirmation dialog centered

PrabhatJTDeveloper committed:
TATB : Make registration page responsive

charmijtdev committed:
- implemented branch

PrabhatJTDeveloper committed:
Merge branch 'feature/authorization-store' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/make-login-page-responsive

PrabhatJTDeveloper committed:
fixed multiple time subscitiont

PrabhatJTDeveloper committed:
Merge branch 'feature/authorization-store' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/make-login-page-responsive

PrabhatJTDeveloper committed:
TATB : Add media query andalso take pull of authentication

charmijtdev committed:
-imported library instead of link
-created seprate route for redirect to exeternal Urls

PrabhatJTDeveloper committed:
TATB : Fix quiz template responsive issue

PrabhatJTDeveloper committed:
TATB - 40 : change design for confirmation dialog box changes - modal component, dialog service

PrabhatJTDeveloper committed:
TATB : Make quiz template header responsive and implement  on hover change bgcolor of dropdown btn

PrabhatJTDeveloper committed:
TATB : Bind state userData with result and login page

PrabhatJTDeveloper committed:
TATB : Fix issue result data is not loading and also unsubscribe observable on destroy

PrabhatJTDeveloper committed:
TATB : bind icon with jyoti technosoft social media profile

PrabhatJTDeveloper committed:
TATB : Add navigation link in header

PrabhatJTDeveloper committed:
TATB : Hide home and result navbar link from login and registration page

charmijtdev committed:
--implemented state for quiz

charmijtdev committed:
Merge branch 'feature/make-login-page-responsive' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/make-login-page-responsive

charmijtdev committed:
Merge branch 'feature/make-login-page-responsive' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/make-login-page-responsive

PoojanPatelJT committed:
Feature/stagging all pr changes (#73)

* Technical assistant : Implement header for all pages

* TATB - 29 : Move inline CSS to external SCSS file

* - TATB 39
    - for wraping dialog data into one config object changed the modal component ts file.
    - in modal component html file use that object .
    - and the modal is opening in signout which is in header component and skip button which is in quiz component so changed this two components.
    - add one object signoutModal in dialogData.json file.

* TATB - 38 : Remove Auth service use to Auth guard for authentication

* TATB - 40 : change design for confirmation dialog box.

* TATB - 29 : Remove inline css and Remove popup box model in header section

* TATB - 29 :  Fixed Merge Conflicts

* TATB - 38 : Remove Auth service use to Auth guard for authentication changes-auth guards,login,header and question service name to chnage authentication service.

* TATB - 40 : change design for confirmation dialog box changes- modal component , login

* TATB-32- fixed conflict in Add footer section in all page

* TATB -35 :Show only recently completed result then create option for showing all results.
  -[BUG] Getting 2 results of same quiz
  -Change the card design of all results page
  -[BUG] Data does not remain after refresh
  -Get data of completed quiz
  -implement initial letter of username as display picture
  -implement show more button in all results

* TATB-32- Fixed conflict in Redesign login and registration page

* TATB - 36 - Fix the issue of save register user data

* TATB - 38 : Remove Auth service use to Auth guard for authentication changes - auth guard,login component,header, authentication service.

* - TATB 39
     - removed ?.length from the login component ts file.
     - removed import of dilogData.json from dialog service
     - updated modal component html file.

* TATB-32- Change class name and  and remove href in Footer section

* Store Implementing with ngrx

* TATB - 40 : change design for confirmation dialog box - change login and registration components

* TATB - 38 : Remove Auth service use to Auth guard for authentication.  changes - result component and authentication service

* TATB-32- change class name and remove inline css in Add footer section

* TATB-29-Add droupdown menu in header section

* TATB-36-Add validation in login page

* - TATB 39
    - removed inline css and give the extenal css
    - added the userId in the login component ts file
    - removed unnecassary line space and comma from the dialog service ts file
    - formated the code in modal component html file and also removed commented code.

* - TATB 39
    - Removed extra comma from the login and header component ts file and also change the function name for opening dialog modal in header component ts file.
    - change the class name in header component scss file.
    - make change in dialogData.json file

* TATB-29-Add header section  in main dashboard and Remove css Add to boostarp class

* - TATB 47
     - have change the validation of password in login component ts file and in html also.

* TATB-29 : Remove unwanted code

* TATB - 38 : bug when user not login and no cookies found redirect to login page

* - TATB 39
   - Modify dialogData.json
   - make code documentation in scss file.
   - as change the json file also updated the html file.

* removed confiict

* - TATB 47
   - make change the html and ts file of login and registration

* - TATB 46
   - add base path in tsconfig.json file
   - and change the path of imports of  all the ts file from ../ to @app or @assets

* TATB-36 : Change href tag to router link  chnage value and Add keydown function in login and registration page

* TATB - 44 Redesign Dashboard page for quiz App

* TATB Stagging : Merge TATB 29 header branch

* TATB -35 :Show only recently completed result then create option for showing all results.
    - change the way of getting result
    -remove unnecessary changes
    -add query params

* TATB - 40 change design for confirmation dialog box

* TATB - 43 : State Management through NGRX
    -Implemented the store for login
    -implemented actions , effect , model , user for login
    -get data from login in state management

* TATB - 40 chnage design for confirmation dialog box

* TATB - 40 change design for confirmation dialog box changes modal component html file.

* TATB - 44 : Redesign Dashbord page for quiz App changes- dashboard componet html file

* TATB - 38 -bug - when user not login and no cookies found redirect to login page changes - app component.html

* TATB - 44 - Redesign Dashboard page for quiz App changes - data.json file,style.scss

* TATB : Fix issue rename emailId to email

* --added store for whole registration

* TATB : Make login page responsive

* TATB : Make footer sticky and also make responsive registration page

* TATB : Change header avatar icon

* TATB : Add background color to footer section

* TATB : Malke confirmation dialog centered

* TATB : Make registration page responsive

* - implemented branch

* fixed multiple time subscitiont

* Merge branch 'feature/authorization-store' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/make-login-page-responsive

* TATB : Add media query andalso take pull of authentication

* -imported library instead of link
-created seprate route for redirect to exeternal Urls

* TATB : Fix quiz template responsive issue

* TATB : Make quiz template header responsive and implement  on hover change bgcolor of dropdown btn

* TATB : Bind state userData with result and login page

* TATB : Fix issue result data is not loading and also unsubscribe observable on destroy

* TATB : bind icon with jyoti technosoft social media profile

* TATB : Add navigation link in header

* TATB : Hide home and result navbar link from login and registration page

* TATB : Fix the issue of checking and validating mobile no.

---------

Co-authored-by: Poojan Patel  <poojan.jtdev@gmail.com>

PrabhatJTDeveloper committed:
Merge branch 'feature/make-login-page-responsive' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/development

PrabhatJTDeveloper committed:
TATB-40-change-design-for-confirmation-dialog-box
